File: programming files/datetime_convert.java
//Bugs Introduced - LH 
import java.time.LocalDateTime;
import java.time.format.DateTimeFormatter;

public class datetime_convert {
    public static void main(String[] args) {
        //LH This is the required date format 
        String dateStr = "2022-03-17 10:45:30";
        DateTimeFormatter formatter = DateTimeFormatter.ofPattern("MM-dd-yyyy HH:mm:ss");
        LocalDateTime dateObj = LocalDateTime.parse(dateStr, formatter);
        String formattedDate = dateObj.format(DateTimeFormatter.ofPattern("yyyy/MM/dd HH:mm:ss"));

        System.out.println(formattedDate);
    }
}

File: programming files/datetime_convert.py
#bugs introduced - LH
from datetime import datetime

date_str = "2022-03-17 10:45:30"

#parsing string into a datetime object
date_obj = datetime.strptime(date_str, '%Y/%m/%d %H::%M::%S') 

#formatting datetime object into mmddyy and hour minute second
formatted_date = date_obj.strftime('%Y/%d/%m %H:%M:%s')

print(formatted_date)
